specs/phase1/beacon-chain.md

```diff
@@ -46,9 +47,11 @@
     - [`get_updated_gasprice`](#get_updated_gasprice)
     - [`get_start_shard`](#get_start_shard)
     - [`get_shard`](#get_shard)
-    - [`get_next_slot_for_shard`](#get_next_slot_for_shard)
+    - [`get_latest_slot_for_shard`](#get_latest_slot_for_shard)
+    - [`compute_offset_slots`](#compute_offset_slots)
```

Comment:
Nitpick: it should be part of "Misc" functions family instead of "Beacon state accessors".